drydock/templates/drydock/task/mongodb/init

```diff
@@ -1,3 +1,6 @@
+{% if 'mongodb-job' in DRYDOCK_INIT_JOBS_EXCLUDED %}
+echo "Skipping MongoDB initialization..."
```

Comment:
You don't need to add this conditional to the template. Just include `mongodb-job` in `DRYDOCK_INIT_JOBS_EXCLUDED` and it should ignore the job

Reply:
Fair point. removed